Cargo.toml

```diff
@@ -38,13 +39,15 @@ systemd-zbus = "0.1"
 # ToDo: set to 0.3 once we support rust 1.63
 time = { version = "=0.3.19", features = ["formatting"] }
 tokio = "1"
+uuid = "1.4.0"
```

Comment:
currently our strategy is to set only major + minor. are you aware of any reasons to don't so?